reasoner/common/src/main/java/com/antgroup/openspg/reasoner/common/graph/vertex/impl/VertexId.java

```diff
@@ -33,13 +34,16 @@ public class VertexId implements IVertexId {
    */
   private final byte[] vertexIdBytes;
 
+  @Getter private String bizId;
```

Comment:
We cannot add the bizId here as this class represents a segment of binary data used as a fixed-length key for RocksDB. Adding the bizId here would result in data from RocksDBGraphState being unable to deserialize the bizId, rendering the entire engine system inoperable.